src/Avatar/MethodInvocation.cs

```diff
@@ -43,17 +43,17 @@ public MethodInvocation(object target, MethodBase method, IArgumentCollection ar
             if (method.GetParameters().Length != Arguments.Count)
                 throw new TargetParameterCountException(ThisAssembly.Strings.MethodArgumentsMismatch(method.Name, method.GetParameters().Length, Arguments.Count));
 
-            callBase = (m, n) => throw new NotImplementedException(ThisAssembly.Strings.CallBaseNotImplemented(ToString()));
+            implementation = (m, n) => throw new NotImplementedException(ThisAssembly.Strings.NotImplemented(ToString()));
         }
 
         /// <summary>
         /// Initializes the <see cref="MethodInvocation"/> with the given parameters.
         /// </summary>
         /// <param name="target">The target object where the invocation is being performed.</param>
         /// <param name="method">The method being invoked.</param>
-        /// <param name="callBase">Delegate to invoke the base method implementation for virtual methods.</param>
-        public MethodInvocation(object target, MethodBase method, ExecuteHandler callBase)
-            : this(target, method, callBase, new ArgumentCollection())
+        /// <param name="implementation">Delegate to invoke the method implementation.</param>
```

Comment:
Is this documentation comment for `implementation` intentionally different from e.g. `MethodInvocation.Create`? There you kept the words "base [method implementation] for virtual methods", you dropped those here.